content/cookbook/01-next/23-chat-with-pdf.mdx

```diff
@@ -22,13 +22,14 @@ Some language models like Anthropic's Claude Sonnet 3.5 and Google's Gemini 2.0
 Create a route handler that will use Anthropic's Claude model to process messages and PDFs:
 
 ```tsx filename="app/api/chat/route.ts"
+import { anthropic } from '@ai-sdk/anthropic';
 import { convertToModelMessages, streamText, type UIMessage } from 'ai';
 
 export async function POST(req: Request) {
   const { messages }: { messages: UIMessage[] } = await req.json();
 
   const result = streamText({
-    model: 'openai/gpt-4o',
+    model: anthropic('claude-sonnet-4-20250514'),
```

Comment:
please just update the string with the right gateway model id (no anthropic import)